examples/lbfgs/driver.py

```diff
@@ -73,27 +70,29 @@ def set_weights(new_weights):
   def net_reinitialization(net_vars):
     return net_vars
 
-  # Create a reusable variable for the network.
+  # Register the network with Ray and create a reusable variable for it.
+  ray.register_class(LinearModel)
```

Comment:
register_class shouldn't be neccessary, we shouldn't have to serialize the network (and we shouldn't, if we have to there is a problem somewhere)